src/main/java/Duke.java

```diff
@@ -1,10 +1,191 @@
+                System.out.println("      [D][ ] " + task + " (by: " + dates[number - 1] + ")");
+                System.out.println("    Now you have " + number + " tasks in the list");
+                System.out.println(DIVIDER);
+                line = in.nextLine();
+            } else if (line.contains("event")) {
+                try {
+                    if (!line.contains("/at")) {
+                        throw new Exception();
+                    }
+                } catch (Exception e) {
+                    System.out.println(DIVIDER);
+                    System.out.println("    ☹ OOPS!!! The description of a event must have '/at' to indicate time.");
+                    System.out.println(DIVIDER);
+                    line = in.nextLine();
+                    continue;
+                }
+                int taskEnd = line.indexOf('/');
+                String task = line.substring(6,taskEnd);
+                dates[number] = line.substring(taskEnd + 4);
+                taskType[number] = "E";
+                lists[number++] = task;
+                System.out.println(DIVIDER);
+                System.out.println("    Got it. I've added this task: ");
+                System.out.println("      [E][ ] " + task + " (by: " + dates[number - 1] + ")");
+                System.out.println("    Now you have " + number + " tasks in the list");
+                System.out.println(DIVIDER);
+                line = in.nextLine();
+            } else if (line.contains("delete")) {
+                try {
+                    int taskDelete = Integer.parseInt(line.substring(7));
+                } catch (StringIndexOutOfBoundsException e) {
+                    System.out.println(DIVIDER);
+                    System.out.println("    ☹ OOPS!!! The description of a delete cannot be empty.");
+                    System.out.println(DIVIDER);
+                    line = in.nextLine();
+                    continue;
+                }
+                int taskDelete = Integer.parseInt(line.substring(7));
+                System.out.println(DIVIDER);
+                System.out.println("    Noted. I've removed this task:");
+                System.out.print("      [" + taskType[taskDelete - 1] + "][" + doneTasks[taskDelete - 1] + "] " + lists[taskDelete - 1]);
+                if(taskType[taskDelete - 1].equals("D")) {
+                    System.out.println(" (by: " + dates[taskDelete - 1] + ")");
+                } else if (taskType[taskDelete - 1].equals("E")) {
+                    System.out.println(" (at: " + dates[taskDelete - 1] + ")");
+                } else {
+                    System.out.println(" ");
+                }
+                System.out.println("    Now you have " + number-- + " tasks in the list.");
+                for (int i = taskDelete - 1; i < number - 1; i++) {
+                    lists[i] = lists[i + 1];
+                }
+                System.out.println(DIVIDER);
+                line = in.nextLine();
```

Comment:
Do create your different classes like Task, Todo, Event, Deadline etc. You are required to implement OOP for these features